lib/gitlab/client/todos.rb

```diff
@@ -0,0 +1,44 @@
+class Gitlab::Client
+  # Defines methods related to todos
+  # @see https://docs.gitlab.com/ce/api/todos.html 
+  module Todos
+    # Gets a list of todos.
+    #
+    # @example
+    #   Gitlab.todos
+    #   Gitlab.todos(action: 'assigned')
+    #   Gitlab.todos(state: 'pending')
```

Comment:
I know it's generally not idiomatic Ruby to include the curly braces but we generate the CLI/Shell help from the YARD so could you include them here?  For the CLI/Shell we require hashes to be passed as valid YAML.